src/Sql/Sql/ChangeLog.md

```diff
@@ -28,7 +28,14 @@
    - Remove-AzSqlInstanceKeyVaultKey
    - Get-AzSqlInstanceTransparentDataEncryptionProtector
    - Set-AzSqlInstanceTransparentDataEncryptionProtector
-
+* Support Azure SQL Server\Database Auditing with new cmdlets.
```

Comment:
@bashahee you'll need to update this file to move the lines that you've added to be under the `## Upcoming Release` header